Src/Tools/DiscoveryDocPatcher/Program.cs

```diff
@@ -69,14 +66,14 @@ static void PatchDirectory(string rootPath)
                 return;
             }
             // Strongly-type properties that are defined without typing.
-            patcher.Replace("schemas.User.properties.addresses", "{ 'type': 'any', 'description': 'Addresses of User'}", "{ 'type': 'array', 'items': { '$ref': 'UserAddress' }, 'description': 'Addresses of User' }");
-            patcher.Replace("schemas.User.properties.emails", "{ 'type': 'any', 'description': 'Emails of User' }", "{ 'type': 'array', 'items': { '$ref': 'UserEmail' }, 'description': 'Emails of User' }");
-            patcher.Replace("schemas.User.properties.externalIds", "{ 'type': 'any', 'description': 'The external Ids of User *' }", "{ 'type': 'array', 'items': { '$ref': 'UserExternalId' }, 'description': 'The external Ids of User *' }");
-            patcher.Replace("schemas.User.properties.ims", "{ 'type': 'any', 'description': 'User&s Instant Messenger' }", "{ 'type': 'array', 'items': { '$ref': 'UserIm' }, 'description': 'User&s Instant Messenger' }");
-            patcher.Replace("schemas.User.properties.organizations", "{ 'type': 'any', 'description': 'Organizations of User' }", "{ 'type': 'array', 'items': { '$ref': 'UserOrganization' }, 'description': 'Organizations of User' }");
-            patcher.Replace("schemas.User.properties.phones", "{ 'type': 'any', 'description': 'Phone numbers of User' }", "{ 'type': 'array', 'items': { '$ref': 'UserPhone' }, 'description': 'Phone numbers of User' }");
-            patcher.Replace("schemas.User.properties.relations", "{ 'type': 'any', 'description': 'The Relations of User *' }", "{ 'type': 'array', 'items': { '$ref': 'UserRelation' }, 'description': 'The Relations of User *' }");
-            patcher.Replace("schemas.User.properties.websites", "{ 'type': 'any', 'description': 'Websites of User' }", "{ 'type': 'array', 'items': { '$ref': 'UserWebsite' }, 'description': 'Websites of User' }");
+            patcher.Replace("schemas.User.properties.addresses", "{ 'type': 'any', 'description': 'A list of the user&s addresses. The maximum allowed data size for this field is 10Kb.'}", "{ 'type': 'array', 'items': { '$ref': 'UserAddress' }, 'description': 'A list of the user&s addresses. The maximum allowed data size for this field is 10Kb.' }");
```

Comment:
Should `user&s addreses` be `user's addresses`? Or is it really broken in the Discovery doc? (It looks like there's a bunch more of these. If it's wrong in the Discovery doc, can we patch it to be more sensible while we're here?

Reply:
In the patcher, we replace single quotes with double quotes so that we don't have to escape double quotes here, which means that we have to use another symbol for actual single quotes, and that's &, hoping we will never get an actual & in the discovery doc :).
https://github.com/googleapis/google-api-dotnet-client/blob/ef2899664b06d0010b2ae81564a6d7f094efcb1b/Src/Tools/DiscoveryDocPatcher/Patcher.cs#L100

Reply:
Aha... that's interesting, as I suspect we *may* see `&` in the discovery docs. I would have thought we'd be able to use something less common such as `¬`. But never mind :)